src/guides/v2.3/frontend-dev-guide/layouts/xml-instructions.md

```diff
@@ -78,6 +78,9 @@ Sample of usage in the product listing page layout:
 <block class="Magento\Catalog\Block\Product\ListProduct" name="category.products.list" as="product_list" template="Magento_Catalog::product/list.phtml"/>
 ```
 
+{:.bs-callout-warning}
+In layout xml when define new block or re-used exists block with `<block></block>` if `template` attribute not available, so that block will not render by default.
```

Comment:
Sorry but the English syntax in this sentence is very jumbled. I've read it several times and I'm not sure what it is trying to say. If you could explain it more fully I can then probably help fix the sentence (eg in English you would never follow the word 'when' with a present tense verb and there are no instances of the word 'the' in this sentence and I think there should be at least 4)

Sorry to be strict but documentation needs to be clear and unambiguous.